R/skim_print.R

```diff
@@ -35,10 +32,22 @@ print.skim_df <- function(x, include_summary = TRUE, n = Inf, width = Inf,
 #' @export
 print.one_skim_df <- function(x, n = Inf, width = Inf, n_extra = NULL, ...) {
   variable_type <- paste("Variable type:", attr(x, "skim_type"))
-  with_line <- cli::rule(line = 1, left = variable_type)
-  print(with_line)
-  out <- format(x, n = n, width = width, n_extra, ...)
-  cat(out[c(-1, -3)], sep = "\n")
+  top_line <- cli::rule(line = 1, left = variable_type)
+  out <- format(x, n = n, width = width, n_extra = n_extra, ...)
```

Comment:
This link (36) I think may be erroring because `...` is used where it shouldn't be.  I'm not sure if that could be part of the issue with the meta data @michaelquinn32 .